build.ps1

```diff
@@ -21,34 +21,36 @@ The build script target to run.
 The build configuration to use.
```

Comment:
How do you update this script (a new version from cake? from Julien's work?)

Reply:
Copy from Julien's work